lib/msal-browser/src/custom_auth/core/network_client/http_client/FetchHttpClient.ts

```diff
@@ -23,7 +23,7 @@ export class FetchHttpClient implements IHttpClient {
     ): Promise<Response> {
         const headers = options.headers as Record<string, string>;
         const correlationId =
-            headers?.[AADServerParamKeys.CLIENT_REQUEST_ID] || undefined;
+            headers?.[AADServerParamKeys.CLIENT_REQUEST_ID] || "";
```

Comment:
Should we create one here rather than use ""

Reply:
We don't need to fallback to a new correlation id here as we rely on one provided by the server.